docs/_guide/decorators.md

```diff
@@ -0,0 +1,260 @@
+*   <code>[queryAssignedNodes](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#queryAssignedNodes)</code>. Create a property getter that returns the children assigned to a specific slot. 
+
+
+All of the decorators can be imported directly from the <code>lit-element</code> module.
+
+```js
+import {eventOptions} from 'lit-element';
+```
+
+
+
+## Other document updates
+
+<<<The following sections belong  in the individual chapters.>>>
+
+
+### customElement
+
+<<<Maybe add this to the to-be-written Writing Components Overview>>>
+
+Use the `customElement` decorator on your component class to define a custom element:
+
+```js
+@customElement('my-element)
+export class MyElement extends LitElement { ... }
+```
+
+The `customElement` decorator takes the place of calling `customElements.define`.
+
+
+### Setting event listener options
+
+<<<add this to the events chapter, probably a new H2 section after "Where to add your event listeners.">>>
+
+When you add an event listener imperatively, using `addEventListener`, you can specify various event listener options. For example, to use a capture phase event listener in plain JavaScript you'd do something like this:
+
+```js
+someElement.addEventListener('click', this._clickHandler, {capture: true});
+```
+
+The `eventOptions` decorator allows you to add event listener options to a listener that's added declaratively in your template.
+
+```js
+@eventOptions({capture: true})
+_clickHandler() { ... }
+
+render() { 
+  return html`
+    <button @click=${this._clickHandler}>Click me!</button>
+  `;
+}
+```
+
+The object passed to `eventOptions` is used as the `options` parameter to `addEventListener`.
+
+More information:
+
+*   [EventTarget.addEventListener()](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) on MDN for a description of the event listener options.
+
+
+### property and internalProperty
+
+<<<The properties chapter already includes property, need to integrate the description of `internalProperty` there ... Unless this is a sufficiently rare use case that it doesn't need to be in the main Properties guide? >>>
+
+Use the `property` and `internalProperty` decorators to declare properties (instead of the static `properties` property).
+
+Use the `property` decorator to declare properties:
+
+```js
+@property({type: String})
+mode = 'auto';
+
+@property({attribute: false})
+data = {};
+```
+
+Use `internalProperty` for private or protected properties that should trigger an update cycle. Properties declared with `internalProperty` shouldn't be referenced from outside the component.
+
+```
+@internalProperty()
+_active = false;
```

Comment:
Is there a reason to use `_active` instead of:

```suggestion
private active = false;
```

I guess guide is more expansive than just TypeScript. It might be fun to use the new private properites syntax.

```suggestion
#active = false;
```

Reply:
Yeah, changed this to `protected active`. Going to avoid the more novel #private syntax for now.